assets/js/modules/analytics/common/account-create.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * AccountCreate component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useState, Fragment, useCallback } from '@wordpress/element';
+import { getQueryArg } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../../../components/button';
+import ProgressBar from '../../../components/progress-bar';
+import { trackEvent } from '../../../util';
+import TimezoneSelect from './timzezone-select';
+import AccountField from './account-field';
+import PropertyField from './property-field';
+import ProfileField from './profile-field';
+
+// import Data from 'googlesitekit-data';
+// const { dispatch, select } = Data;
+
+const AccountCreate = () => {
+	// Uncomment this section after 1311 is merged.
+	/*
+	const isDoingCreateAccount = useSelect(
+		( select ) => {
+			return select( STORE_NAME ).isDoingCreateAccount();
+		},
+		[]
+	);
+	const accountTicketTermsOfServiceURL = useSelect(
+		( select ) => {
+			return select( STORE_NAME ).getAccountTicketTermsOfServiceURL();
+		},
+		[]
+	);
+
+	*/
+	//const { createAccount } = useDispatch( STORE_NAME );
+
+	/*
+	// Redirect if the accountTicketTermsOfServiceURL is set.
+	if ( accountTicketTermsOfServiceURL ) {
+		location = accountTicketTermsOfServiceURL;
+	}
+	*/
+
+	const isDoingCreateAccount = false;
+	const { createAccount } = () => {};
+	const { siteName, siteURL, timezone: tz } = global.googlesitekit.admin;
+	const errorCode = getQueryArg( location.href, 'error_code' );
+
+	// Handle expected provisioning flow error codes.
+	let errorMessage = false;
+	switch ( errorCode ) {
+		case 'user_cancel':
+			errorMessage = __( 'The Terms of Service were not accepted.', 'google-site-kit' );
+			break;
+
+		case 'max_accounts_reached':
+			errorMessage = __( 'The Google Analytics account limit has been reached.', 'google-site-kit' );
+			break;
+
+		case 'backend_error':
+			errorMessage = __( 'Unknown service error.', 'google-site-kit' );
+			break;
+	}
+	const [ error, setError ] = useState( errorMessage );
```

Comment:
This shouldn't be necessary. There's a `getError()` selector in the `modules/analytics` datastore that will hold the API error if the `createAccount` action failed.

Potentially we should incorporate the logic to parse the `error_code` query arg to an error (should be an object with `code` _and_ `message`) into a resolver for `getError`, so that this is handled in the datastore, rather than in the component.

Reply:
Currently handling the error at the proxy so I have removed this section of the code.